crates/router/src/core/errors.rs

```diff
@@ -461,3 +461,45 @@ pub enum WebhooksFlowError {
     #[error("Dispute webhook status validation failed")]
     DisputeWebhookValidationFailed,
 }
+
+#[cfg(feature = "detailed_errors")]
```

Comment:
Optional change: You could move these new types and impls to a module to simplify the feature gates.